generators/client/templates/react/src/main/webapp/app/app.tsx.ejs

```diff
@@ -16,6 +16,7 @@ import AppRoutes from './routes';
 export interface IAppProps {
   location: any;
   isAuthenticated?: boolean;
+  isAdmin: boolean;
```

Comment:
I would like to do it differently so that role management is generic. So let me update this part